src/styles/variables.scss

```diff
@@ -4,8 +4,8 @@
  */
 
 // font defaults
-$fontSizeDefault: 1.2rem; // = 12px
-$fontStack: roboto, sans-serif;
+$fontSizeDefault: 1rem; // = 10px
+$fontStack: "Noto Sans", sans-serif;
```

Comment:
Oh I did notice that this made the legend text a bit smaller (1.2rem/12px), it would probably look better setting the legend text to either 1.4rem or 1.6rem IMO.

Reply:
Also seeing @unixjazz 's comment about the `<h2>` font-size being smaller then the `<p>` size in the `<header>`. Previously the h2 elements were 18px/1.8rem in size.